src/store/index.ts

```diff
@@ -1,15 +1,16 @@
 import { createStore, applyMiddleware, Store } from 'redux';
 import { logger } from '../middleware';
 import rootReducer, { RootState } from '../reducers';
+import {composeWithDevTools} from "redux-devtools-extension";
```

Comment:
Please keep conventions. whitespace between curly braces.

+ It'd be better importing vendor-module first. (place it to line 2)

Reply:
Should be fine now, I will have to change settings of my IDE for auto-imports ;)